packages/atomic/src/components/result-lists/result-list-common.tsx

```diff
@@ -195,26 +195,21 @@ export class ResultListCommon {
     return window.innerHeight + window.scrollY >= childEl?.offsetHeight;
   }
 
-  public componentDidRender(
-    firstSearchExecuted: boolean,
-    listWrapperRef?: HTMLDivElement
-  ) {
-    if (firstSearchExecuted) {
-      listWrapperRef?.classList.remove('placeholder');
-    }
-  }
-
```

Comment:
When the result list loads/render is not a good indicator that the results load, cause the template components files have to load too. This caused a big gap on slow connections